### PR TITLE
Connect project to the API

### PR DESCRIPTION
**Pull request by RKDGT**
add usage for api 
fix some issues

### CODE REVIEW

File: src/components/getFromApi/GetReviewListOfHotel.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import axios from './AxiosConfig'
+
+////id - of object 
+//// loc -location 
+const GetReviewsListOfHotel = (args) =>{
+    return axios.get('https://hotels4.p.rapidapi.com/reviews/list',{
```
Comment (IraCherginyshka) [inline src/components/getFromApi/GetReviewListOfHotel.jsx]: I can't find AxiosConfig.
You need to create a folder "services" and move this file

File: src/components/getFromApi/GetReviewListOfHotel.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+           params: {
+                id: args
```
Comment (IraCherginyshka) [inline src/components/getFromApi/GetReviewListOfHotel.jsx]: Do we need to set locale?

File: src/components/getFromApi/GetReviewListOfHotel.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import axios from './AxiosConfig'
+
+////id - of object 
+//// loc -location 
+const GetReviewsListOfHotel = (args) =>{
```
Comment (IraCherginyshka) [inline src/components/getFromApi/GetReviewListOfHotel.jsx]: You can use 
= ({ id }) => 
Let me know if you have any questions.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

File: src/components/hotelsInCity/HotelInCityItem.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState} from 'react'
+import CardItem from '../cards/CardItem'
+import GetDetails from '../getFromApi/GetDetails'
+
+const HotelInCityItem = (props) => {
```
Comment (IraCherginyshka) [inline src/components/hotelsInCity/HotelInCityItem.jsx]: = ({ id, name }) => {

File: src/components/hotelsInCity/HotelsInCity.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
+.hotels{
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  background: #fff;
+}
+
+
+
```
Comment (IraCherginyshka) [inline src/components/hotelsInCity/HotelsInCity.css]: delete

File: src/components/hotelsInCity/HotelsInCity.css [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+.hotels-items-container{
+  width: 100%;
+  justify-content: 10px;
+}
+
+.hotels-item-pic-wrap{
+  width: 100px;  
+  height: 200px;
+}
+
```
Comment (IraCherginyshka) [inline src/components/hotelsInCity/HotelsInCity.css]: delete line

File: src/components/getFromApi/GetReviewListOfHotel.jsx (hunk 1 above)
Comment (RKDGT) [reply src/components/getFromApi/GetReviewListOfHotel.jsx]: i'm create folder "services" and move file here

File: src/components/getFromApi/GetReviewListOfHotel.jsx (hunk 2 above)
Comment (RKDGT) [reply src/components/getFromApi/GetReviewListOfHotel.jsx]: we discussed in team and as result we resolve to add this feature in future
(means yes we need to set locale)

File: src/components/getFromApi/services/AxiosConfig.jsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+
+export default axios.create ({
+    headers: {
+        'x-rapidapi-key': '06680bfbb0msh40b9b35c3e0b9ebp1c368fjsn857cece02668',
+        'x-rapidapi-host': 'hotels4.p.rapidapi.com',
+        useQueryString: true
+    }
+})
```
Comment (IraCherginyshka) [inline src/components/getFromApi/services/AxiosConfig.jsx]: u can add this 
 axios.create({
  baseURL: 'https://hotels4.p.rapidapi.com',
});
 u don't need to repeat it every time

Comment (IraCherginyshka) [reply src/components/getFromApi/services/AxiosConfig.jsx]: Move all files to /src/services folder

File: src/components/hotelsInCity/GetHotelImages.jsx [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    console.log(id)
+    GetHotelsPhotoById({id: id})
+    .then(res => {
+        const pictInfo = []
+        console.log(res.data.hotelImages)
+        res.data.hotelImages.map(info => pictInfo.push({baseUrl: info.baseUrl, sizes: info.sizes}))
+        const srcOnPict =[]
+        pictInfo.map(info => 
+            info.sizes.map(size => srcOnPict.push(info.baseUrl.replace('{size}',size.suffix))))
+        console.log(srcOnPict)
```
Comment (IraCherginyshka) [inline src/components/hotelsInCity/GetHotelImages.jsx]: delete all console.log

File: src/components/hotelsInCity/HotelsInCity.jsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  useEffect(()=>
+  SearchByKeyWord({query: props.name})
+  .then(res => {
+    setAppState(res.data.suggestions[1].entities);
+  })
+  .catch(error => {
+    console.log(error)  
+  }),[setAppState]
+  )
+  console.log(appState);
```
Comment (IraCherginyshka) [inline src/components/hotelsInCity/HotelsInCity.jsx]: delete

File: src/services/GetHotelsPhotoById.jsx [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import axios from './AxiosConfig'
+
+////ARI url 
+////id - of Hotel
+const GetHotelsPhotoById = ({id}) => {
```
Comment (IraCherginyshka) [inline src/services/GetHotelsPhotoById.jsx:5]: 👍

File: src/services/GetReviewListOfHotel.jsx [hunk 11]
```diff
@@ -0,0 +1,11 @@
+import axios from './services/AxiosConfig'
+
+////id - of object 
+const GetReviewsListOfHotel = ({id}) =>{
+    return axios.get('https://hotels4.p.rapidapi.com/reviews/list',{
+           params: {
+                id: args
```
Comment (IraCherginyshka) [inline src/services/GetReviewListOfHotel.jsx]: args??? maybe id?

Comment (RKDGT) [reply src/services/GetReviewListOfHotel.jsx]: ups

**Review by IraCherginyshka — APPROVED** [review]

**Review by mskab — APPROVED** [review]

**Pull request merged**